api.swagger.yaml

```diff
@@ -1355,27 +1355,108 @@ paths:
+            items:
+              type: string
+              format: uuid
+        - description: 'Filter groups that where created after (greater than or equal) the timestamp (in seconds).'
+          in: query
+          name: created_start_timestamp
+          schema:
+            type: integer
+            example: 1563357496
+        - description: 'Filter groups that where created before (less than or equal) the timestamp (in seconds).'
+          in: query
+          name: created_end_timestamp
+          schema:
+            type: integer
+            example: 1563357496
+        - description: 'Filter groups with last posts after (greater than or equal) the timestamp (in seconds).'
+          in: query
+          name: last_post_start_timestamp
+          schema:
+            type: integer
+            example: 1563357496
+        - description: 'Filter groups with last posts before (less than or equal) the timestamp (in seconds).'
+          in: query
+          name: last_post_end_timestamp
+          schema:
+            type: integer
+            example: 1563357496
+        - description: 'Filter groups on the privacy setting. Multiple values may be supplied.'
+          in: query
+          name: privacy_setting
+          schema:
+            type: array
+            items:
+              type: string
+              enum:
+                - Public
+                - Private
+                - Secret
+        - description: 'Filter groups with the membership types. Multiple values may be supplied.'
+          in: query
+          name: members_type
+          schema:
+            type: array
+            items:
+              type: string
+              enum:
+              - Administrators
+              - Members
+              - None
+        - description: 'Filter groups with name changes after (greater than or equal) the timestamp (in seconds).'
+          in: query
+          name: name_change_start_timestamp
+          schema:
+            type: integer
+            example: 1563357496
+        - description: 'Filter groups that have had new administrators after (greater than or equal) the timestamp (in seconds).'
+          in: query
+          name: new_admin_start_timestamp
+          schema:
+            type: integer
+            example: 1563357496
+        - description: 'Filter groups that have had privacy setting changes after (greater than or equal) the timestamp (in seconds).'
+          in: query
+          name: privacy_setting_change_start_timestamp
+          schema:
+            type: integer
+            example: 1563357496
         - description: Field to sort by.
           in: query
           name: sort
           schema:
             type: string
             enum:
-              - relevance
+              - group_name
               - likes
               - members_count
+              - members_type
+              - post_count
+              - privacy_setting
+              - relevance
```

Comment:
default sort should be created date, switch to relevance for searches

Reply:
done